### PR TITLE
Pioneer DDJ-FLX4: Added controller mapping documentation

### PR DESCRIPTION
**Pull request by Robert904**
Documentation for the proposed mapping for Pioneer controller DDJ-FLX4.

Preview:
https://deploy-preview-534--mixxx-manual.netlify.app/hardware/controllers/pioneer_ddj_flx4

PR for the mapping is https://github.com/mixxxdj/mixxx/pull/11245

### CODE REVIEW

**Comment by Robert904** [comment]
Not sure what should be done... The errors reported by the link checker are for files not modified by this PR...

**Comment by JoergAtGithub** [comment]
> Not sure what should be done... The errors reported by the link checker are for files not modified by this PR...

The link issues are fixed in the 2.3 branch now. To get these fixes, call:
```
git fetch upstream
git merge upstream/2.3
```

File: source/_static/controllers/pioneer_ddj_flx4.svg
Comment (Swiftb0y) [inline source/_static/controllers/pioneer_ddj_flx4.svg:1]: Could you please remove all the brand-logos and text from the drawings? I think they carry a copy-right infringement risk with them.

**Review by Swiftb0y — COMMENTED** [review]
Manual looks good otherwise

**Comment by Robert904** [comment]
Just pushed a new version with a new mapping for the FX buttons, inline with the modification done in the DDJ-400 mapping

**Comment by Robert904** [comment]
Retargetted at 2.4

**Comment by ronso0** [comment]
Looks like rebase went wrong, this `Merge branch 'ddj_flx4' into 2.4`, too.
Please rebase interactively onto 2.4 and remove all unrelated commits.

**Comment by Robert904** [comment]
I actually avoided using rebase to avoid problems (I had to look for retarget procedure in internet;..), so I forced content of 2.4 into my branch and reapplied my commits... (sorry if it is uselessly complex and if my commit message was not good...)

The resulting branch compared to the 2.4 is indeed only my 7 added files !

Is it not good like that ? (I was busy doing the same for the mapping itself...)

**Comment by ronso0** [comment]
Well, here it's 23 commits: 8 about the FLX4 plus merge commits and unrelated stuff.

I don't see any actual inline code review that might get detached during rebase, so just go ahead and rebase I'd say 🤷‍♂️

**Comment by Robert904** [comment]
OK... ried again, I hope I did not make a mess!

**Comment by Robert904** [comment]
And removed the brands/logos as requested

**Comment by Robert904** [comment]
broken checks are not linked to this PR...

**Comment by ronso0** [comment]
Okay, great. It's just 5 commits now (compared to 8 I saw before), did you double-check the chapter includes everything you added earlier?
https://deploy-preview-534--mixxx-manual.netlify.app/hardware/controllers/pioneer_ddj_flx4

**Comment by Robert904** [comment]
I just realized the page numbers pointing to the manufacturer's manual were not valid anymore, I updated them.
Other than that, looks good to me !

**Comment by Holzhaus** [comment]
> Now just waiting for the mapping itself to be merged :-)

Don't worry, we usually wait until both the mapping itself and the corresponding documentation in the manual are ready and then merge both at the same time :smile:

**Review by Swiftb0y — APPROVED** [review]

**Review by ronso0 — COMMENTED** [review]
Some small requests

**Comment by ronso0** [comment]
Looks like I was too slow with my review...

@Robert904 Do you mind taking a look at my suggestions nonetheless and maybe implement them in a follow-up PR?

**Comment by Swiftb0y** [comment]
Sorry merging prematurely... 🫣

**Comment by ronso0** [comment]
@Swiftb0y don't worry, I started this review yesterday and didn't manage to finalize it until now. maybe should have posted it chunk by chunk.

**Pull request merged**